ui/src/views/device-profile-templates/CreateDeviceProfileTemplate.tsx

```diff
@@ -24,37 +24,39 @@ function CreateDeviceProfileTemplate() {
     });
   };
 
-  const codecScript = `// Decode uplink function.
-  //
-  // Input is an object with the following fields:
-  // - bytes = Byte array containing the uplink payload, e.g. [255, 230, 255, 0]
-  // - fPort = Uplink fPort.
-  // - variables = Object containing the configured device variables.
-  //
-  // Output must be an object with the following fields:
-  // - data = Object representing the decoded payload.
-  function decodeUplink(input) {
-    return {
-      data: {
-        temp: 22.5
-      }
-    };
-  }
-  
-  // Encode downlink function.
-  //
-  // Input is an object with the following fields:
-  // - data = Object representing the payload that must be encoded.
-  // - variables = Object containing the configured device variables.
-  //
-  // Output must be an object with the following fields:
-  // - bytes = Byte array containing the downlink payload.
-  function encodeDownlink(input) {
-    return {
-      bytes: [225, 230, 255, 0]
-    };
-  }
-  `;
+  const codecScript = `/**
+ * Decode uplink function
+ * 
+ * @param {object} input
+ * @param {number[]} input.bytes Byte array containing the uplink payload, e.g. [255, 230, 255, 0]
+ * @param {number} input.fPort Uplink fPort.
+ * @param {Record<string, string>} input.variables Object containing the configured device variables.
+ * 
+ * @returns {{data: Record<string, any>}} Object representing the decoded payload.
```

Comment:
Thanks for pushing the fix, but I think this applies to all input / return data types isn't it?